sdk/nodejs/__tests__/cloudClient.spec.ts

```diff
@@ -115,22 +115,22 @@ describe('DVCCloudClient without EdgeDB', () => {
         it('throws an error for a null event', () => {
             const badEvent = null
             expect(async () => await client.track(user, badEvent as unknown as DVCEvent))
```

Comment:
I don't think the async and await are needed in these tests. The underlying function returns a promise, which `expect()` can handle directly